modules/travel_pay/app/services/travel_pay/claims_service.rb

```diff
@@ -21,31 +21,37 @@ def get_claims(params = {})
       }
     end
 
-    def get_claims_by_date_range(params = {})
-      validate_date_params(params['start_date'], params['end_date'])
+    def get_claims_by_date_range(params = {}) # rubocop:disable Metrics/MethodLength
+      DateTime.parse(params['start_date'].to_s) && DateTime.parse(params['end_date'].to_s)
```

Comment:
Looks like this line only exists to trigger a side effect in a somewhat unlikely case. It trips me up because it's essentially a no-op for the common case. 

Is there anyway we can keep the `validate_date_params` method (or call it `validate_date_range` or something) and stick lines 25 and 45-51 in it? That way, we at least know the purpose of the date parsing without assignment.

Reply:
✅